.github/pull_request_template.md

```diff
@@ -1,42 +1,20 @@
-<!-- Thank you for submitting a pull request! Find more information in our development guide at https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md and contact us on #dev in Slack. -->
-
+<!--Thank you for submitting a pull request! Find more information in our development guide at https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md and contact us on #dev in Slack. -->
+<!--Provide an overview of the PR for maintainers and reviewers.-->
 ## Description
 
-<!-- Elaborate beyond the title of the PR as necessary to help the reviewers and maintainers.-->
-
-<!-- Answer the following questions to help reviewers and maintainers
-understand this PR's scope at a glance:
--->
-
-> Is this change a fix, improvement, new feature, refactoring, or other?
-
-> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)
-
-> How would you describe this change to a non-technical end user or system administrator?
-
-## Related issues, pull requests, and links
 
-<!-- List any issues fixed by this PR, and provide links to other related PRs, upstream release notes, and other useful resources. For example:
-* Fixes #issuenumber
-* Related documentation in #issuenumber
-* [Some release notes](http://usefulinfo.example.com)
--->
 
-<!-- The following sections are filled in by the maintainer with input from the contributor:
-Use :white_check_mark: or (x) to signal selection.
--->
+<!--Provide a user-friendly explanation, keep it brief if it isn't user-visible.-->
+## Non-technical explanation
 
-## Documentation
 
-( ) No documentation is needed.
-( ) Sufficient documentation is included in this PR.
-( ) Documentation PR is available with #prnumber.
-( ) Documentation issue #issuenumber is filed, and can be handled later.
 
+<!--Mark the appropriate option with an (x). Propose a release note if you can.-->
 ## Release notes
 
-( ) No release notes entries required.
-( ) Release notes entries required with the following suggested text:
+( ) This is not user-visible and no release notes are required.
+( ) Release notes are required, please propose a release note for me.
```

Comment:
is it intentionally not using checkboxes (`* [ ]`)?

checkboxes would allow the option to be marked with a click.

Reply:
I think @mosabua told me at one point that that something in Trino's Github automation expects all of them to be ticked before a PR is merged?

Reply:
Correct.. if you use [] it counts as a build action..

Reply:
I edited description of https://github.com/trinodb/trino/pull/1394 so it includes checkboxes.
@colebow @mosabua where can i see the consequences of the fact these checkboxes are not checked yet?